target/linux/ath79/image/generic.mk

```diff
@@ -171,6 +171,24 @@ define Device/comfast_cf-e120a-v3
 endef
 TARGET_DEVICES += comfast_cf-e120a-v3
 
+define Device/comfast_cf-wr650ac-v1
+  ATH_SOC := qca9558
+  DEVICE_TITLE := COMFAST CF-WR650AC v1
```

Comment:
Please update to use DEVICE_VENDOR, DEVICE_MODEL and DEVICE_VARIANT as introduced in:
https://github.com/openwrt/openwrt/commit/8c3dbaf8b49233e275b2dce4e30dc0a2bf8c86c5

Reply:
fixed